src/components/common/inputs/date-input.jsx

```diff
@@ -7,16 +7,11 @@ import ValidationError from '../section/validation-error';
 
 export default class DateInput extends React.Component {
   handleUserInput(e, inputName) {
-    let value = e.target.value;
-    if (inputName === 'time' && value === '') {
-      value = null;
-      e.target.value = null;
-    }
-    this.props.onChange(inputName, value);
+    this.props.onChange(inputName, e.target.value || null);
   }
 
   render() {
-    let className = 'x-date';
+    let className = 'col-of-two x-date';
```

Comment:
Making styles for date and time inputs work for various screen sizes:
![date-inputs-resizing](https://user-images.githubusercontent.com/15594382/151745024-be149a01-ddc6-4d80-8e1b-9afb9a8fa82e.gif)